_rules/element-lang-valid-de46e4.md

```diff
@@ -238,6 +238,30 @@ This `div` element has an invalid `lang` [attribute value][]. The [accessible na
 </html>
 ```
 
+#### Failed Example 8
+
+The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code, hence not a [valid language tag][].
```

Comment:
How about making this a little more explicit:

```suggestion
The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code. This is not part of the [language subtag registry][].
```

Reply:
I think we should stick with the "not a valid tag" explanation as we do for grandfathered tags (https://github.com/act-rules/act-rules.github.io/pull/1650/files#r673783251) (for the sake of consistency). WDYT?
```suggestion
The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code. This is not a [valid language tag][].
```